src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -344,24 +344,24 @@ public void testContaminationFilter() throws Exception {
 
 
         final Set<VariantContext> variantsFilteredAtZeroPct =
-                StreamSupport.stream(new FeatureDataSource<VariantContext>(filteredVcfNoContamination).spliterator(), false)
+                Utils.streamVcf(filteredVcfNoContamination)
                 .filter(vc -> vc.getFilters().contains(GATKVCFConstants.CONTAMINATION_FILTER_NAME))
```

Comment:
It might be worth indenting these two lines to be consistent with the other similar lines in this file.

Reply:
done